### ISSUE TITLE
business continuity

### ISSUE THREAD

**Issue by ItsCloudyOutThere** [ilyas-it83/CloudComparer#222]
Hi,

Amazing list, very helpful to see all services listed.
I had one question, something in my area of expertise.

Under business continuity you list AWS Workspaces and AWS AppStream together with various other solutions.
As this is more EUC focused, still business continuity though, AppStream and Workspaces should be in the same category. Workspaces, right now, is in the Office Document solution space and it really is a DaaS offering, delivering a desktop from the cloud. If it was to be places with AppStream it would fit. (It could also have its own row)

That having said, once that is done, in the Microsoft Column Azure Virtual Desktop could be added as well on the same row as Workspaces and AppStream is. (RDP as protocol)

Huawei also has a desktop solution called FusionCloud (protocol HDP)

Jus thought this would make it more complete.

**Comment by ilyas-it83** [comment]
Makes sense @ItsCloudyOutThere, would you mind raising this change as a PR to merge.

**Comment by ilyas-it83** [comment]
This issue is stale because it has been open for 30 days with no activity.
